.github/workflows/ubuntu_build.yml

```
@@ -0,0 +1,66 @@
+name: Calculix 2.17 with preCICE adapter test build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, ubuntu-20.04]
+        include:
+          - os: ubuntu-18.04
+            name: bionic
+          - os: ubuntu-20.04
+            name: focal
+#          - os: ubuntu-21.04
+#            name: hirsute
+    runs-on: ${{matrix.os}}
+    continue-on-error: true
+
+    steps:
+    - name: Update system
+      run: sudo apt update && sudo apt upgrade -y
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.ref }}
+    - name: get preCICE
```

Tiny consistency thing:
```suggestion
    - name: Get preCICE
```
(also below)

To make it clearer for the future, I would include the preCICE version in the name:

```suggestion
    - name: Get preCICE v2.3.0
```